open_bounties/0052 Shopify NFT App.md

```diff
@@ -0,0 +1,81 @@
+---
+ID: "0052"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0052
+Category: eCommerce App
+Status: Open
+Amount: $15,000
+---
+
+<!-- Please update this title -->
+
+# Shopify NFT App
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+There are over 2.5 million merchants that use the Shopify platform. All of which use marketing in some form or another to drive sales. 
+
+The goal of this app will be to give merchants a way to increase engagement through the use of NFTs. The actual marketing method is left up to the developer and the only requirement for this bounty is to use NFTs to incentivize customers to make further purchases or to increase engagement with the brand.
+
+Example:
+A customer would claim a random NFT via a QR code on their physical receipt or a link on their order confirmation page. Then upon collecting a certain set of NFT's that customer would trade them in for a discount or a free product.
+
+Such an app would allow merchants to use NFTs in their business, strengthen their communities and explore what Web3 has to offer. It would also expose them to  the speed and efficiency of using XRPL.
+
+
+
+## Milestones
+
+<!--
+Please split the bounty into smaller milestones with individual awards in the following template.
+The first milestone should be the core functionality, while the rest can be useful add-ons.
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| 1 | ... | ... | $... |
+
+(The proposed amounts from milestones should add up to the amount listed at the top of the bounty proposal)
+-->
+
+| # | Description | Details | Proposed Potential Award |
+| --- | --- | --- | --- |
+|1| Create | Create a Shopify app that uses XLS-20 NFTs on XRPL to drive sales and engagement for a merchant. Detailed requirements are listed below.| $5,000 |
+|2| Launch | Submit and launch the app on the Shopify app store | $5,000 |
+|3| Adoption | Get 100+ installs | $5,000 |
+
+**Detailed requirements:**
+Merchant dashboard features
+- [ ] merchant XRPL account setup
+- [ ] create/view/delete XLS-20 NFT on XRPL
+  - Create campaign with properties  
+  - [ ] allow specification of start date, end date and redemption limits
+  - [ ] associate reward for NFT such as discount, BOGO, free product
+  - [ ] included products or collections
+  - [ ] selection for NFTs used in the campaign
+  - [ ] qualifying conditions and (ie. must have 3 specific NFTs)
```

Comment:
Seems like there's a missing phrase after `and` here.

Reply:
Fixed